kolibri/plugins/management/assets/src/device_management/views/manage-content-page/task-progress.vue

```diff
@@ -74,6 +75,13 @@
         required: true,
       },
       id: RequiredString,
+      cancellable: {
+        type: Boolean,
+        required: true,
+      },
+    },
+    watch: {
+      taskHasFailed: 'emitFailure',
```

Comment:
I would just inline `emitFailure` here, if it's not used by anything else in the component.